### PR TITLE
Free vs paid usage limits, audit log retention period config

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Per https://seqera.slack.com/archives/C07K4GWNR6E/p1729535796959649 

Also closes https://seqera.atlassian.net/browse/EDU-238

### CODE REVIEW

**Review by justinegeffen — APPROVED** [review]
Thank you!

File: platform_versioned_docs/version-24.1/limits/limits.mdx [hunk 1]
```diff
@@ -9,37 +9,37 @@ Seqera Platform elements and features have default limits per workspace and orga
...
-| Datasets     | 100   |
-| Labels       | 1000  |
+| Description  | Free | Cloud Pro + Enterprise |
+| ------------ | ---- | ---------------------- |
+| Active runs  | 5    | 100, or per license    |
+| Members      | 50   | 50, or per license     |
+| Participants | 50   | 50, or per license     |
+| Pipelines    | 100  | 100, or per license    |
+| Datasets     | 100  | 1000, or per license   |
+| Labels       | 1000 | 1000, or per license   |
```
Comment (JaimeSeqLabs) [inline platform_versioned_docs/version-24.1/limits/limits.mdx:19]: For enterprise, the code sets the default max amount of labels per workspace to 100 unless a limit is set in the license.
By default, new licenses are created with a 1000 limit for labels. Is that the default you are referring to?

File: platform_versioned_docs/version-24.1/limits/limits.mdx [hunk 2]
```diff
@@ -9,37 +9,37 @@ Seqera Platform elements and features have default limits per workspace and orga
...
-| ------------ | ----- |
-| Active runs  | 5     |
-| Members      | 50    |
-| Participants | 50    |
-| Pipelines    | 100   |
-| Datasets     | 100   |
-| Labels       | 1000  |
+| Description  | Free | Cloud Pro + Enterprise |
+| ------------ | ---- | ---------------------- |
+| Active runs  | 5    | 100, or per license    |
```
[the remark this answers is not in the document]
Comment (llewellyn-sl) [reply platform_versioned_docs/version-24.1/limits/limits.mdx:14]: @JaimeSeqLabs this is the default limit we set for Enterprise licenses, unless custom limits are agreed upon. @starnsd to keep me honest.

File: platform_versioned_docs/version-24.1/limits/limits.mdx (hunk 1 above)
Comment (llewellyn-sl) [reply platform_versioned_docs/version-24.1/limits/limits.mdx:19]: Yes, referring to the default license limit here. There's also a note on the page that mentions that older licenses may have a limit of 100 and people should contact support for an updated license if needed.

**Pull request merged**